playbooks/byo/openshift-etcd/config.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Create initial host groups for localhost
```

Comment:
Please use the openshift-cluster/initialize_groups.yml include here instead of running these tasks.  It will flag this as an entry point and run the syntax-check in tox tests.

For example: https://github.com/openshift/openshift-ansible/blob/master/playbooks/byo/openshift-etcd/restart.yml#L2